src/main/java/microsoft/exchange/webservices/data/core/ExchangeServiceBase.java

```diff
@@ -484,12 +484,6 @@ protected void setCustomUserAgent(String userAgent) {
    * @throws microsoft.exchange.webservices.data.exception.ServiceLocalException the service local exception
    */
   public void validate() throws ServiceLocalException {
```

Comment:
maybe the method can be declared as `abstract` since it does not contain any default implementation anymore.